spec/features/user/sign_in_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.feature 'Sign in', type: :feature, focus: true, js: true do
```

Comment:
- [ ] Should be `feature 'Users'`
- [ ] Context should be `Sign In`
- [ ] Remove ` focus: true, js: true`